lib/utils/forkedblockchain.js

```diff
@@ -56,7 +57,57 @@ function ForkedBlockchain(options) {
   } else {
     this.fork = options.fork;
   }
+
   this.forkBlockNumber = options.fork_block_number;
+  this.forkCacheSize = parseInt(options.fork_cache_size == null ? 10000 : options.fork_cache_size);
```

Comment:
Should we default to `10000`? I wonder if this should be opt in, though I don't suspect this will interfere with any current tests.